### ISSUE TITLE
KL Loss Weights

### ISSUE THREAD

**Issue by kakusikun** [gaosh/Structured-Bayesian-Pruning-pytorch#1]
Thanks for porting tensorflow to pytorch, it's a great work.

https://github.com/necludov/group-sparsity-sbp/blob/master/nets/metrics.py#L49
The author does not set kl weight ( sum up the KL loss and use default weight 1 ) ( or I didn't find it out?)

https://github.com/gaosh/Structured-Bayesian-Pruning-pytorch/blob/master/LeNet.py#L111
you set kl loss weights as [0.3, 0.3, 0.2, 0.2], how do you set it?

I have been tried SSL before, the weights are similar to regularization parameter which could influence the final sparsity of model.

**Comment by gaosh** [comment]
Yeah, in original paper they didn't modified the weights of kl loss. I aim to test the influence of the weights of kl loss, you can change it back or any other values you want.

**Comment by kakusikun** [comment]
OK, it seems that the parameters tuning is inevitable. 
Thanks for your reply.
